examples/timeseries/timeseries_transformer_classification.py

```diff
@@ -0,0 +1,189 @@
+"""
+Title: Time Series Classification with a Self-Attention Transformer Model
+Authors: [Theodoros Ntakouris](https://github.com/ntakouris)
+Date created: 2021/06/25
+Last modified: 2021/06/25
+Description: This notebook demonstrates how to do timeseries forecasting using a transformer model.
+"""
+
+
+"""
+##
+This is the [Attention Is All You Need Paper](https://arxiv.org/abs/1706.03762), but for time series instead of natural language processing.
```

Comment:
This example would benefit from an introduction and a conclusion that highlights benefits (or weaknesses) of using this architecture for timeseries classification compared to other available options.